lib/utils.pm

```diff
@@ -1375,6 +1376,16 @@ sub get_root_console_tty {
     return (sle_version_at_least('15') && !is_caasp) ? 6 : 2;
 }
 
+=head2 get_x11_console_tty
+    Returns tty number used designed to be used for X
+    Since SLE 15 gdm is always running on tty7, currently the main GUI session
+    is running on tty2 by default. see also: bsc#1054782
+=cut
+sub get_x11_console_tty {
+    my $new_gdm = !(is_sle && !sle_version_at_least('15')) && !(is_leap && !leap_version_at_least('15'));
+    return (check_var('DESKTOP', 'gnome') && get_var('NOAUTOLOGIN') && $new_gdm) ? 2 : 7;
```

Comment:
This broke CaaSP-controller test which is basically SLE12_SP2 support_server but with DISTRI=caasp, DESKTOP=gnome, NOAUTOLOGIN=1 set ---> $new_gdm = True

 it returns **2** but it should be **7**

Reply:
should be fixed by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3677